src/features/components/templates/footer/index.tsx

```diff
@@ -26,11 +29,9 @@ export const MainFooter = () => {
         setIsCareer(is_career_page)
     }, [])
 
-    const socialButtons = is_career
-        ? socialButtonsCareers
-        : is_eu
-        ? socialButtonsEU
-        : socialButtonsROW
+    const socialButtons = useMemo(() => {
+        return getSocialButtons(lang, is_eu, is_career)
+    }, [is_eu, lang, is_career])
```

Comment:
Due to the fn calls only once, we have no updated object. Let's try this one

```suggestion
    let socialButtons = {}
    useEffect(() => {
    socialButtons = getSocialButtons(lang, is_eu, is_career)
    }, [is_eu, lang, is_career])
```